cfg/config.go

```diff
@@ -49,17 +53,21 @@ var (
 // NewDefaultConfig retrieves the config with sane defaults
 func NewDefaultConfig() *Configuration {
 	return &Configuration{
-		MountPath:               "/data",
-		BackupAnnotation:        "k8up.syn.tools/backup",
-		BackupCommandAnnotation: "k8up.syn.tools/backupcommand",
-		FileExtensionAnnotation: "k8up.syn.tools/file-extension",
-		ServiceAccount:          "pod-executor",
-		BackupCheckSchedule:     "0 0 * * 0",
-		GlobalKeepJobs:          6,
-		BackupImage:             "172.30.1.1:5000/myproject/restic",
-		PodExecRoleName:         "pod-executor",
-		RestartPolicy:           "OnFailure",
-		MetricsBindAddress:      ":8080",
-		PodFilter:               "backupPod=true",
+		MountPath:                   "/data",
+		BackupAnnotation:            "k8up.syn.tools/backup",
+		BackupCommandAnnotation:     "k8up.syn.tools/backupcommand",
+		FileExtensionAnnotation:     "k8up.syn.tools/file-extension",
+		ServiceAccount:              "pod-executor",
+		BackupCheckSchedule:         "0 0 * * 0",
+		GlobalKeepJobs:              6,
+		BackupImage:                 "172.30.1.1:5000/myproject/restic",
+		PodExecRoleName:             "pod-executor",
+		RestartPolicy:               "OnFailure",
+		MetricsBindAddress:          ":8080",
+		PodFilter:                   "backupPod=true",
+		GlobalCPUResourceRequest:    "100m",
+		GlobalCPUResourceLimit:      "250m",
+		GlobalMemoryResourceRequest: "128Mi",
+		GlobalMemoryResourceLimit:   "256Mi",
```

Comment:
@Kidswiss does it make sense to provide global defaults? What are usually wrestic's requirements? or should we leave that empty? (I think there shouldn't be a CPU limit by default for example)